src/components/CharacterLimit/CharacterLimit.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import { render } from '@testing-library/react';
+
+import CharacterLimit from './CharacterLimit';
+
+describe('CharacterLimit', () => {
+  it('renders CharacterLimit component', () => {
+    const { container } = render(
+      <CharacterLimit value="Hello, world!" maxLength={140} />
```

Comment:
O value nao existe aqui. O character limit nao recebe ele como props.

```suggestion
      <CharacterLimit maxLength={140} />
```